apps/els_lsp/test/els_code_action_SUITE.erl

```diff
@@ -156,3 +161,28 @@ fix_module_name(Config) ->
     ],
   ?assertEqual(Expected, Result),
   ok.
+
+-spec remove_unused_macro(config()) -> ok.
+remove_unused_macro(Config) ->
+  Uri = ?config(code_action_uri, Config),
+  Range = els_protocol:range(#{from => {?COMMENTS_LINES+17, 9}, to => {?COMMENTS_LINES+17, 15}}),
```

Comment:
We will probably get a lint error here (spaces around +) in CI.

Reply:
addressed in a3d3b9a